src/main/scala/tech/cryptonomic/conseil/util/ConfigUtil.scala

```diff
@@ -0,0 +1,37 @@
+package tech.cryptonomic.conseil.util
+
+import com.typesafe.config.Config
+import tech.cryptonomic.conseil.generic.chain.MetadataTypes.{Network, Platform}
+
+import scala.collection.JavaConverters._
+
+object ConfigUtil {
```

Comment:
Please consider that we now have typed configurations from the conf file parsed in the Main apps, that uses pureconfig to create such objects.

You might want to integrate Utils with such configs

Reply:
Yes, I'm aware of that, I'll update my branch with master and then I'll make changes